### DISCUSSION TITLE
Support status for `embedded:network/interface/wifi`?

### DISCUSSION THREAD

**Discussion by cmidgley** [Moddable-OpenSource/moddable#1416] [Q&A] [answered]
After not realizing that I should have used `io` and now having to port off the legacy websocket, I'd like to avoid that mistake again.  I need to implement `WiFi`, and see that ECMA-419 has `embedded:network/interface/wifi`.  Searching 5.0.0 doesn't find any implementation.  Could you confirm that's the case and that I should proceed with implementation on the non-IO version?

**Comment by phoddie** [comment, marked as answer]
You are correct: there isn't an implementation of [this 2nd Edition module](https://419.ecma-international.org/#-17-network-interface-class-pattern-wi-fi-network-interface) available yet. It somehow slipped through the cracks. My 419 focus at the moment is on 3rd Edition, but will add this to the to-do list. It shouldn't be too bad, though the unreliable and asynchronous nature of Wi-Fi tends to make it more complicated to get right than it might seem.

**Comment by cmidgley** [comment]
Thanks for the clarification.  I'm not dependent upon it - I can use the legacy WiFi for now and migrate later to 3rd edition if/when it becomes available.
